inst/rmarkdown/templates/Brattle_HTML/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,38 @@
+---
+title: "Document Title"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    number_sections: false
+    theme: united
+    fig_width: 8
+    fig_height: 6
+    fig_caption: true
+    df_print: kable
+    code_folding: hide
+---
+
+***
+
+*PRIVILEGED AND CONFIDENTIAL*
+
+*PREPARED AT THE REQUEST OF COUNSEL*
+
+***
+
+# Overview
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      warning = FALSE,
+                      message = FALSE)
+
+library(pacman)
```

Comment:
One way to do a check for this is this little snippet:
if(!require(pacman)) install.packages("pacman")